### ISSUE TITLE
Full screen Google Ads set up screen/page

### ISSUE THREAD

**Issue by findingsimple** [woocommerce/google-listings-and-ads#80]
**Description:**

Using the scaffolding created in https://github.com/woocommerce/google-listings-and-ads/issues/44 set up a full-screen google ads onboarding flow. (If in doubt these 2 flows should look and act consistently.

At the top left of the page will be a backlink with the text "< Set up paid campaign" -  this will return the user to the dashboard (TBC)

At the top right of the page will be a help link pointing to extension documentation (location TBC).

---

(Updated 22 March 2021)

The setup process will consist of 3 steps:

1. Set up your accounts
2. Configure your paid campaign
3. Setup Billing

---

Each of these is essentially a tabbed content area (or equivalent approach).

- Start at Step 1.
- Step 2 is only available once Step 1 is completed (i.e. all accounts are connected/created).
- Step2 text etc. is not clickable (disabled) until step 1 is completed
- Once "Set up your accounts" is complete the number "1" becomes a blue checkmark and content from step 2 is displayed. it becomes clickable so that merchants can go back to step 1 if they wish (and vice versa). If they then disconnect an account I guess step 2 becomes unavailable again.
- At the top of each step is a label (Step 1, Step2), Heading (Set up your accounts), and a brief description of the step.

**Note** - during development, we may need to fake some of this logic initially (or options etc.) to avoid being blocked - actual functionality can be added as it is finalized.

---

**Screenshots**

Step 1

![Ads set up 1](https://user-images.githubusercontent.com/355014/100405476-9c084900-30b3-11eb-83c9-cc423d3cc1f7.png)

Step 2

![Ads budget1](https://user-images.githubusercontent.com/355014/100405481-9e6aa300-30b3-11eb-907a-79d2594a450c.png)

---

**Technical**

* Will likely need to be able to "link" to steps (just something for consideration)

---

**Acceptance criteria:**

* [ ] Logic above is followed
* [ ] Button colors updates to reflect admin profile color scheme
* [ ] Tracks event fired on clicking the back link, help button or continue button
* [ ] Translatable text

---

**Dependencies**

* [Full page merchant center setup screen/page](https://github.com/woocommerce/google-listings-and-ads/issues/44)

**Out of bounds/rabbit holes**

* For the Initial implementation lets just use a static placeholder image for the mock product listing on step 2- we'll look at dynamically generating mock product listings based on store products as a nice to have in a follow-up issue.
* Looking into the more complicated elements of the setup screens e.g. connect to wordpress, connect to google etc. etc. should be handled in separate PRs/issues - this is primarily about getting the page to load as a first step

**Event tracking:**

* [ ] Tracks event fired on clicking the back link, help button or continue button

---

Part of https://github.com/woocommerce/google-listings-and-ads/issues/20
